avro2tf/src/main/scala/com/linkedin/avro2tf/constants/Avro2TFJobParamNames.scala

```diff
@@ -26,4 +26,5 @@ object Avro2TFJobParamNames {
   final val ENABLE_TERM_ONLY_FEATURE_LIST = "enable-term-only-feature-list"
   final val DISCARD_UNKNOWN_ENTRIES = "discard-unknown-entries"
   final val ENABLE_FILTER_ZERO = "enable-filter-zero"
+  final val PASS_THROUGH = "pass-through"
```

Comment:
maybe better to call it "pass-through-only" to differentiate pass-through-columns in the future if we want to have

Reply:
sure